package.json

```diff
@@ -55,12 +60,6 @@
     "react-hot-loader": "4.12.18",
     "svg-inline-loader": "0.8.0",
     "terser-webpack-plugin": "2.3.0",
-    "tslint": "5.20.1",
-    "tslint-config-prettier": "1.18.0",
-    "tslint-eslint-rules": "5.4.0",
-    "tslint-plugin-prettier": "2.0.1",
-    "tslint-react": "4.1.0",
-    "tslint-react-hooks": "2.2.1",
```

Comment:
Are the hook rules preserved with ESlint? They are incredibly useful when working with hooks :)